.prettierrc

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Can you remove values set here that are already defaults? [List of options and defaults here, just ctrl+f through it.](https://prettier.io/docs/en/options.html)